src/codegen/mod.rs

```diff
@@ -3299,7 +3298,7 @@ pub fn codegen(context: &mut BindgenContext) -> Vec<P<ast::Item>> {
         }
 
         context.resolve_item(context.root_module())
-            .codegen(context, &mut result, whitelisted_items, &());
+            .codegen(context, &mut result, codegen_items, &());
```

Comment:
File a follow up `E-Easy` issue to remove this argument, and remove the `whitelisted_items` parameter from all the codegen functions using `ctx.codegen_items()` instead?